admin/featured_categories.php

```diff
@@ -25,7 +25,7 @@
         case 'setflag':
             if (isset($_POST['flag']) && ($_POST['flag'] === '1' || $_POST['flag'] === '0')) {
                 zen_set_featured_status((int)$_POST['id'], (int)$_POST['flag']);
-                zen_redirect(zen_href_link(FILENAME_FEATURED_CATEGORIES, zen_get_all_get_params(['action', 'fID']) . 'fID=' . $_POST['id'], 'NONSSL'));
+                zen_redirect(zen_href_link(FILENAME_FEATURED_CATEGORIES , zen_get_all_get_params(['action' , 'fID']) . 'fID=' . $_POST['id'] , 'NONSSL'));
```

Comment:
Here (and a couple of other places 'downstream' in the code) you've added an unwanted space _before_ a function parameter's separating comma.  Line 28 was just fine (and proper) the way it was; please revert these.

There should be a space _after_ each parameter-separating comma, not before.